### DISCUSSION TITLE
Pulsar auto delete policy

### DISCUSSION THREAD

**Discussion by river1017** [apache/pulsar#22697] [Q&A]
I changed the configuration in pulsar's configuration file broker.conf to

brokerDeleteInactiveTopicsEnabled: true
brokerDeleteInactiveTopicsMode: delete_when_no_subscriptions
brokerDeleteInactiveTopicsMaxInactiveDurationSeconds: 120

Then I restarted pulsar's docker container and sent a message to the topic "persistent://t_test/proj_test/web_hook_test", but when time passed, the topic was not removed and the admin rest api could query the topic information.
This topic does not have subscribers and the data returned by queries through the interface "/admin/v2/persistent/t_test/proj_test/web_hook_test/subscriptions" is empty

**Comment by lhotari** [comment]
There's a limitation in the deleting inactive topics when it comes to completely unused topics. If the namespace bundle of the topic isn't active, the topics won't get deleted. 

It looks like there's an issue #22717 matching this question. Continuing on the issue.
